projects/aca-shared/src/lib/services/aca-mobile-app-switcher.service.ts

```diff
@@ -112,4 +120,11 @@ export class AcaMobileAppSwitcherService {
   getCurrentUrl(): string {
     return window.location.href;
   }
+
+  closeDialog(): void {
+    if (this.dialogRef !== null && this.dialogRef !== undefined) {
```

Comment:
the same, can be reduced to just: 
if (this.dialogRef)

Reply:
same done, reduced to single condition now